### PR TITLE
Who To Follow And Adjustments to Feed Events

### PR DESCRIPTION
**Pull request by kespinola**
## Changes
- [X] [Feature] Feed event objects have fields for the source wallet and their profile.
- [X] [Fix] Write follow events for the from_account not the to_account
- [X] [Feature] Add `wallet_totals` table for writing number of followers and following for a wallet using a db trigger on `graph_connection` insert
- [x] [Feature] Query for getting recommend wallets to follow based on the `wallet_totals`

### CODE REVIEW

**Review by kristianeboe — APPROVED** [review]

File: crates/core/src/db/queries/feed_event.rs [hunk 1]
```diff
@@ -25,14 +25,16 @@ use crate::{
 /// join of event tables into a single event type
 pub type Columns<'a> = (
     models::FeedEvent,
+    String,
+    Option<String>,
```
Comment (ray-kast) [inline crates/core/src/db/queries/feed_event.rs:29]: What are the column names for these two types?  Some descriptive comments like some of the other query modules have would be good

File: crates/graphql/src/schema/objects/wallet.rs [hunk 2]
```diff
@@ -13,6 +13,21 @@ pub struct Wallet {
     pub twitter_handle: Option<String>,
 }
 
+impl TryFrom<(models::WalletTotal, Option<String>)> for Wallet {
```
Comment (ray-kast) [inline crates/graphql/src/schema/objects/wallet.rs:16]: This doesn't appear to fail, and as such should just be Into

File: crates/core/migrations/2022-05-02-113557_create_wallet_totals_table/up.sql [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+);
+
+create index if not exists wallet_totals_following on wallet_totals (following);
+create index if not exists wallet_totals_followers on wallet_totals (followers desc);
+
+insert into wallet_totals (address, following, followers)
+ select
+    address,
+    (select count(from_account) from graph_connections WHERE from_account = wallets.address) as following,
+    (select count(to_account) from graph_connections WHERE to_account = wallets.address) as follwers
```
[the remark this answers is not in the document]
Comment (kespinola) [reply crates/core/migrations/2022-05-02-113557_create_wallet_totals_table/up.sql:14]: Its a small table. not super performant but just needs to run once.

[the remark this answers is not in the document]
Comment (ray-kast) [reply crates/core/migrations/2022-05-02-113557_create_wallet_totals_table/up.sql:14]: ok.  as long as nothing gets stuck on this we should be good, we can just keep an eye on dev to see if it has problems with this.

**Review by ray-kast — APPROVED** [review]

**Pull request merged**